ansible/roles/nginx/templates/connectbox_captive_portal.conf.j2

```diff
@@ -3,8 +3,8 @@ server {
     listen *:80 default_server;
     server_name _;
     index index.html index.htm;
-    access_log {{ connectbox_log_dir }}/captive-portal-access.log;
-    error_log {{ connectbox_log_dir }}/captive-portal-error.log error;
+    access_log {{ connectbox_log_dir }}/captive_portal-access.log;
```

Comment:
Given this vhost is now used for captive portal and chat server, it makes sense to rename these log files, and the host template itself e.g. captive_portal -> app_server

Reply:
hey, thanks for the review :) I think that the way nginx is currently configured, the chat requests are still going to the general connectbox access/error log and only the captive portal requests are going into these files. That's why I left these named as captive_portal